src/reactapp/config-overrides.js

```diff
@@ -24,17 +24,15 @@ module.exports = function override(config, env) {
         name: false,
       },
     },
-    resolve: {
-      ...config.resolve,
-      alias: {
-        ...config.resolve.alias,
-        // Uncomment these to reduce the size of the compiled app
-        // this improves performance, but you loose compatibility
-        // with the React browser extension for debugging
-        //
-        // react: 'preact/compat',
-        // 'react-dom': 'preact/compat',
-      },
-    },
   };
+
+  if (isEnvProduction) {
+    baseConfig.resolve.alias = {
+      react: 'preact/compat',
+      'react-dom': 'preact/compat',
+      ...baseConfig.resolve.alias
```

Comment:
The spreading should appear above in this case. Because we need all settings present there (that will be done by the spreading) then, we need our custom changes. so it should be:

```
 ...baseConfig.resolve.alias,
react: 'preact/compat',
 'react-dom': 'preact/compat',
```

Sorry for not giving clarity before.